kratos/solving_strategies/strategies/butcher_tableau.h

```diff
@@ -66,21 +66,26 @@ namespace Kratos
 template<typename Derived, unsigned int TOrder, unsigned int TSubstepCount>
 class ButcherTableau
 {
+
 public:
+
+    typedef array_1d<double, TSubstepCount> VectorType;
+    typedef array_1d<double,TSubstepCount-1> RowType;
+    typedef std::array<RowType, TSubstepCount-1> MatrixType;
+
     ///@name Type Definitions
     ///@{
```

Comment:
```suggestion
    ///@name Type Definitions
    ///@{

    typedef array_1d<double, TSubstepCount> VectorType;
    typedef array_1d<double,TSubstepCount-1> RowType;
    typedef std::array<RowType, TSubstepCount-1> MatrixType;
```

This way they are in the Type definitions section